haml/_oisc_warning.html.haml

```diff
@@ -0,0 +1,19 @@
+-# OISC Warning
+
+-# Set defaults for parameters
+- title ||= ''
+- body ||= ''
+- is_sticky ||= false
```

Comment:
Is this a good idea? It strikes me as an opportunity for errors (missing params) to be silently swallowed by a consumer of the design-system.

Reply:
This is a good point, I think it's correct for `body` which is optional, and `is_sticky` as a silent default is _intentional_.  But let me test without it for `title`, as that should really throw an error if not provided.